docs/bokeh/switcher.json

```
@@ -1,10 +1,14 @@
 [
   {
     "name": "latest",
-    "version": "3.4.1",
+    "version": "3.5.0",
     "url": "https://docs.bokeh.org/en/latest/",
     "preferred": true
   },
+  {
+    "version": "3.4.1",
+    "url": "https://docs.bokeh.org/en/3.4.1/"
```

```suggestion
    "version": "3.4.2",
    "url": "https://docs.bokeh.org/en/3.4.2/"
```

Not sure what happens to the `3.4.3` point release anyhow the latest point release is not `3.4.1`.

---

thanks for the catch @mosc9575